### ISSUE TITLE
consider indenting children of debug accessible view elements

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#221601]
@jooyoungseo and @rperez030, we now have a debug accessible view. Currently, I'm not indenting the children of elements, but we could. 

What do you think?

**Comment by rperez030** [comment]
Interesting. Indenting may be the only way to convey the structure of a list or a class. The current debug UI with its tree structure is not that bad actually. I wonder how efficient would be to navigate a text-based version. I think I need to give it a try.

**Comment by meganrogge** [comment]
Yes, from my testing, the tree structure in the debug console is highly accessible at the moment. The issue is one cannot navigate it character by character, thus the need for the accessible view. 

The other issue was that variables weren't expanded before. They are now in screen reader mode by default, and one can disable that via `debug.expandLazyVariables`.

Would appreciate your thoughts on the experience. I'm working on improving it as my focus for July.
